HealthcareSystem/Backend/Service/RoomAndEquipment/IEquipmentTypeService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Backend.Model.Manager;
+using System.Collections.Generic;
+
+namespace Backend.Service.RoomAndEquipment
+{
+    public interface IEquipmentTypeService
+    {
+        EquipmentType AddEquipmentType(EquipmentType equipmentType);
+        EquipmentType GetEquipmentTypeById(int id);
+        List<EquipmentType> GetEquipmentTypes();
+        EquipmentType EditEquipmentType(EquipmentType equipmentType);
```

Comment:
Why is it Edit here, and Set in the repository? I think Update was generally used for both.

Reply:
Fixed in 52728b8266cccd1bd69b043a7e6060b01c8ed960